layouts/partials/nav.html

```diff
@@ -20,8 +20,10 @@
 
             <div class="navbar-collapse collapse" id="navigation">
                 <ul class="nav navbar-nav navbar-right">
+                  {{ $current := . }}
                   {{ range .Site.Menus.main }}
-                  <li class="dropdown">
+                  {{ $topLevel := replace .URL "/" "" }}
+                  <li class="dropdown{{ if not $current.IsHome }}{{ if or (eq $current.URL .URL) (eq $current.Type $topLevel) }} active{{ end }}{{ end }}">
```

Comment:
beginning with Hugo v0.55, `Page's .URL is deprecated and will be removed in a future release.`

therefore replace `$current.URL` with `$current.RelPermalink` to avoid warning messages